dune-project

```diff
@@ -14,4 +14,4 @@
  (synopsis "Streaming client for Memprof")
  (description "Generates compact traces of a program's memory use.")
  (depends
-  (ocaml (>= 4.11.0))))
+  (ocaml (or (>= 5.3.0) (and (>= 4.11.0) (< 5.0))))))
```

Comment:
Could this consider the ocaml-variants `5.2.0+statmemprof`?

Reply:
I was treating that variant as an alpha release until 5.3.0 was available. I don't see much appeal in supporting that now that 5.3 is available.